packet/src/ipv6.rs

```diff
@@ -62,9 +62,9 @@ impl Ipv6Hdr {
         v
     }
 
-    pub fn adjust_hlim(pkt: &mut Packet, delta: i8) {
-        let hdr = pkt.hdrs.ipv6_hdr.as_mut().unwrap();
-        hdr.ipv6_hop_lim = (hdr.ipv6_hop_lim as i8 + delta) as u8;
+    pub fn adjust_hlim(pkt: &mut Packet, delta: i16) {
```

Comment:
This is an 8 bit field - why allow a 16 bit delta?

Reply:
It was so I could update it to set it to 0 for a test:

> ipv6::Ipv6Hdr::adjust_hlim(&mut to_send, -255);